components/financial_assistance/app/views/financial_assistance/applications/review.html.erb

```diff
@@ -150,16 +170,86 @@
             </div>
 
             <div class="row row-form-wrapper ptb no-buffer row-height form-content">
-              <div class="col-md-6">Is this person currently enrolled in health coverage?</div>
+              <div class="col-md-6"><%= applicant_currently_enrolled.gsub(' *','')%></div>
               <div class="col-md-6"><%= human_boolean applicant.has_enrolled_health_coverage %></div>
             </div>
 
             <div class="row row-form-wrapper ptb no-buffer row-height form-content">
-              <div class="col-md-6">Does this person currently have access to other health coverage, including through
-                another person?
+              <div class="col-md-6"><%= applicant_eligibly_enrolled.gsub(' *','')%>
               </div>
               <div class="col-md-6"><%= human_boolean applicant.has_eligible_health_coverage %></div>
             </div>
+
+            <% if EnrollRegistry[:indian_health_service_question].feature.is_enabled && applicant.indian_tribe_member %>
```

Comment:
Do it like this `FinancialAssistanceRegistry.feature_enabled?(:indian_health_service_question)`. And make sure to use `FinancialAssistanceRegistry` contstant within the engine